docs/CHANGELOG.rst

```diff
@@ -13,6 +13,10 @@ All notable changes to this project will be documented in this file.
 `Unreleased`_
 =============
 
+Deprecated
+----------
+* Qt version/architecture/modules checking (#807, #812)
+
```

Comment:
Deprecated? I think a deprecated function is still available. We just deleted it, so I suggest using a new word. Which word?

In addition:

```suggestion
Deprecated
----------
* Qt version/architecture/modules checking (#748, #807, #812)

```

.

Reply:
I would like to use "Deleted"